src/Init/Data/Json/Parser.lean

```diff
@@ -0,0 +1,332 @@
+/-
+Copyright (c) 2019 Gabriel Ebner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+
+Authors: Gabriel Ebner, Marc Huisinga
+-/
+prelude
+import Init.Data.Json.Basic 
+import Init.Control.Except
+
+inductive Quickparse.Result (α : Type)
+| success {} (pos : String.Iterator) (res : α) : Quickparse.Result
+| error {} (pos : String.Iterator) (err : String) : Quickparse.Result
+
+def Quickparse (α : Type) : Type := String.Iterator → Quickparse.Result α
+
+def String.Iterator.skipWsCore : Nat → String.Iterator → String.Iterator
+| 0  , it => it
+| i+1, it =>
+  let c := it.curr;
+  if c = '\u0009' ∨ c = '\u000a' ∨ c = '\u000d' ∨ c = '\u0020' then
```

Comment:
```suggestion
  if c.isWhitespace then
```

Reply:
I'm not sure about this change. As far as I can tell, there are plenty of conflicting notions of what a whitespace character is, and the JSON standard is very precise about which one they prefer. I'd rather be explicit about it in the parser, than replace this with Char.isWhitespace, which just so accidentally happens to coincide with the notion of whitespace in JSON.

Reply:
Ok